package/kedro_viz/server.py

```diff
@@ -539,8 +540,8 @@ def nodes_metadata(node_id):
         dataset_metadata = _get_dataset_metadata(node)
         return jsonify(dataset_metadata)
 
-    # return empty JSON for parameters type
-    return jsonify({})
+    parameters_metadata = {"parameters": node["obj"].load()}
```

Comment:
I guess you are going to refactor this to cater for cases like `params:foo.bar`, correct?

Reply:
Yep, I've addressed that in https://github.com/quantumblacklabs/kedro-viz/pull/275/commits/cfa570355fcd58d39adf143f8d076b8a7e05fbf8